src/main/java/sx/blah/discord/handle/obj/IChannel.java

```diff
@@ -764,71 +764,16 @@
 	boolean isDeleted();
 
 	/**
-	 * A permission override for a role or user.
-	 */
-	class PermissionOverride implements IIDLinkedObject {
```

Comment:
This is a breaking change, so you should deprecate this and make the replacement class extend this instead.

Reply:
How is this a breaking change? It's a non-public API, it shouldn't have ever been utilized externally.

Reply:
It's very much a public api. https://github.com/austinv11/Discord4J/blob/caa3476ec433a055b6b31b34cb33c57b9c889a5c/src/main/java/sx/blah/discord/handle/obj/IChannel.java#L618